etc/analytics.api.md

```diff
@@ -14,6 +14,7 @@ export function analytics(config: AnalyticsConfig): AnalyticsEventService;
 export interface AnalyticsConfig {
     authorization: string;
     authorizationType: 'apiKey' | 'bearer';
+    debug?: boolean;

```

Comment:
Do we want to add something to the README about this?

Reply:
It sounds like there are more documentation updates that are needed, maybe we could couple this with those? Iirc you had a list of items for Chris to update in docs, could you add this to that list?

Reply:
Done.